src/main/java/step2/Calculator.java

```diff
@@ -0,0 +1,33 @@
+package step2;
+
+public class Calculator {
+
+  private final static String SPLIT_SPACE = " ";
+
+  public int calculate(String input) {
+
+    if(isEmpty(input)) {
+      throw new IllegalArgumentException();
+    }
+
+    String[] splitInputArr = input.split(SPLIT_SPACE);
+
+    int result = Integer.parseInt(splitInputArr[0]);
+    for (int i = 1; i < splitInputArr.length; i += 2) {
+      String operator = splitInputArr[i];
+      int number = Integer.parseInt(splitInputArr[i+1]);
+      result = getCalculateResult(operator, result, number);
```

Comment:
```suggestion
      CalculateOperator operator = CalculateOperator.valueOfOperator(splitInputArr[i]);
      int number = Integer.parseInt(splitInputArr[i + 1]);
      result = operator.calculate(result, number);
```
계산 가능한 연산자를 찾는건 CalculateOperator에 역할을 맡기는게 좋겠네요.
CalculateOperator가 연산자 상수를 다 가지고 있으니깐요.
CalculateOperator가 연산자에 해당되는 계산식을 알고 있으니 계산에 필요한 값만 전달해서 계산 결과 받아오면 getCalculateResult 메서드를 없앨 수 있어서 코드량을 줄일 수 있습니다.